docs/src/api.md

```diff
@@ -0,0 +1,25 @@
+# APIs
+
+```@docs
```

Comment:
Is there a reason we can't use `@autodocs` here instead of listing the functions below?

Reply:
It's okay to use `@autodocs` here, but without the `@autodocs` we can choose the order of the docstrings and add more descriptions between docstrings in the future. If we missed some docstrings in md file, then Documenter.jl will raise warning about that. (`cis(::Quaternion)` will be removed, so I think we do not need the documentation.)

Reply:
okay, we can do this for now.